import-scripts/cvr_consent_status_checker.py

```diff
@@ -192,6 +197,16 @@ def main():
         sys.exit(2)
 
     expected_consent_status_values = fetch_expected_consent_status_values()
+    if expected_consent_status_values is None:
+        print >> ERROR_FILE, "Unable to fetch consent status values, exiting..."
+        email_consent_status_report(
```

Comment:
In its current state, the code will email something like:

```
Consent status update:


CONSENT STATUS GRANTED:

CONSENT STATUS REVOKED:
```

Wondering if I should make the message more specific